src/ambulance_game/markov/additional.py

```diff
@@ -0,0 +1,224 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import tikzplotlib
+
+from .markov import (
+    build_states,
+    visualise_ambulance_markov_chain,
+)
+
+
+def convert_networkxx_figure_to_tikz(
+    num_of_servers, threshold, system_capacity, parking_capacity
+):
+    """TODO: Build a string of latex code that generates the tikz picture of the networkxx model as constructed by the networkxx library.
+    """
+
+    visualise_ambulance_markov_chain(
+        num_of_servers=num_of_servers,
+        threshold=threshold,
+        system_capacity=system_capacity,
+        parking_capacity=parking_capacity,
+    )
+    tikzplotlib.save("example.tex")
```

Comment:
What is this for?

Reply:
This is an extra function that I may or may not look into more. This one generates latex code for a tikz picture of the networkx graph that is built by python. I realise it's not particularly useful and I may never use it, but thought of maybe having it in anyway. Do you think I should just leave it out?

Reply:
Let's remove it from the code for now: paste it in to a GitHub issue and come back to it when/if you want/need to. Always good to keep the code base clean.

Reply:
Sure thing. Will do